site/gatsby-site/src/utils/AlgoliaUpdater.js

```diff
@@ -193,12 +193,47 @@ class AlgoliaUpdater {
       flag: 1,
     };
 
-    const reports = await this.mongoClient
+    let reports = await this.mongoClient
       .db('aiidprod')
       .collection(`reports`)
       .find({}, { projection })
       .toArray();
 
+    // Only index Incident Reports
+    reports = reports.filter(
```

Comment:
I would prefer to have only one function that decides if a report is an incident report or not, it looks like this should use the "isCompleteReport" function.

Reply:
Done